types/mydecimal.go

```diff
@@ -1518,7 +1518,17 @@ func DecimalSub(from1, from2, to *MyDecimal) error {
 	return doAdd(from1, from2, to)
 }
 
+func resetDecimal(d *MyDecimal) {
+	if d == nil {
+		return
+	}
+	for i := range d.wordBuf {
+		d.wordBuf[i] = 0
```

Comment:
Should the other member attributes of MyDecimal be reset?
We can not be sure whether we'll add any other functions that need to call resetDecimal.

Reply:
What're your opinions about how to reset other attributes in `MyDecimal`?
How about setting them to zero values?

Reply:
That's what I mean.

Reply:
fixed, PTAL.